docs/yaml/functions/run_target.yaml

```diff
@@ -40,7 +40,7 @@ kwargs:
       the first item will find that command in `PATH` and run it.
 
   depends:
-    type: list[build_tgt | custom_tgt]
+    type: list[build_tgt | custom_tgt | custom_idx]
```

Comment:
This needs a note at the end of this block that `custom_idx` support was added in 1.5.0

Reply:
Actually, it was possible before the typing checks (0.60). I have documented it this way.